pkg/operator/resource/resourceapply/generic.go

```diff
@@ -154,6 +155,16 @@ func ApplyDirectly(clients *ClientHolder, recorder events.Recorder, manifests As
 				result.Error = fmt.Errorf("missing apiExtensionsClient")
 			}
 			result.Result, result.Changed, result.Error = ApplyCustomResourceDefinitionV1(clients.apiExtensionsClient.ApiextensionsV1(), recorder, t)
+		case *storagev1.StorageClass:
+			if clients.kubeClient == nil {
+				result.Error = fmt.Errorf("missing kubeClient")
+			}
+			result.Result, result.Changed, result.Error = ApplyStorageClass(clients.kubeClient.StorageV1(), recorder, t)
```

Comment:
where is ApplyStorageClass? Missing file?

Reply:
It already exists, here:

https://github.com/openshift/library-go/pull/815/files/54705a0472e861a45c437f0f8592a0e78dca37ab#diff-3de2a0c77901abef949759ee8869c9d1L21

Reply:
ic